modules/products/bamboo/variables.tf

```diff
@@ -46,6 +46,12 @@ variable "installation_timeout" {
   }
 }
 
+variable "termination_grace_period" {
+  description = "Termination grace period in seconds"
+  type        = number
+  default     = 30
```

Comment:
As we have default in root level, we can remove the default (make it mandatory for module) here and instead we may add a validator to limit the grace period to a logical range. WDYT?

The same comment for this variable in other modules.

Reply:
Yeah, I was thinking about it. Let me do that!